docs/applications/github-action/02-gcp.md

```diff
@@ -1,8 +1,8 @@
 ---
-id: applications-container-repositories-gcp
-slug: /applications/container-repositories/gcp
-title: GCP Container Repositories
-sidebar_label: GCP Container Repositories
+id: applications-github-action-gcp
+slug: /applications/github-action/gcp
```

Comment:
These are just so the pages don't break until I can update the docs for these clouds specifically.